content/chat/getting-started/javascript.textile

```diff
@@ -0,0 +1,195 @@
+---
+title: "Getting started: Chat and JavaScript"
+meta_description: "A getting started guide for Ably Chat that steps through some of the key features using JavaScript."
+meta_keywords: "Ably, realtime, quickstart, getting started, basics, chat"
+---
+
+This guide will get you started with Ably Chat.
+
+It will take you through the following steps:
+
+* Complete the prerequisites before you begin.
+* Create a client and establish a realtime connection to Ably.
+* Create a room and subscribe to its messages.
+* Send a message to the room, and then edit that message.
+* Set up a typing indicator to see which clients are typing.
+* Display the online status of clients to the room.
+* Subscribe and send reactions to the room.
+* Retrieve historical messages to provide context to new joiners.
+
+h2(#prerequisites). Prerequisites
+
+* Sign up for an Ably account.
+* Create a new app, and get your first API key.
+** Your API key will need the @publish@, @subscribe@, @presence@ and @history@ capabilities.
+* Install the Ably CLI: @npm install -g &#64;ably/cli@
+* Run @ably login@ in the terminal and follow the prompts to provide an access token to the Ably CLI.
+* Set the default app to use with @ably apps switch@ and select an API key to use for operations with @ably auth keys switch@.
+* Install the Ably Chat JavaScript SDK in a new project within your IDE. This will also install the Ably Pub/Sub SDK as it is a dependency:
+
+```[sh]
+npm install @ably/chat
+npm install react
+```
+
+h2(#step-1). Step 1: Connect to Ably
+
+Clients establish a connection with Ably when they instantiate an SDK. This enables them to send and receive messages in realtime across channels.
+
+* Run the following function to instantiate a realtime client with the Pub/Sub SDK and then pass that client into the Chat SDK constructor. At the minimum you need to provide an authentication mechanism and a @clientId@ to provide the client with an identity. Use an API key when authenticating for simplicity, but you should use token authentication in a production app:
+
+```[javascript]
+import * as Ably from 'ably';
+import { ChatClient, AllFeaturesEnabled, MessageEvent, Message } from '@ably/chat';
+
+async function getStarted() {
+
+const realtimeClient = new Ably.Realtime({ key: '{{API_KEY}}', clientId: 'my-first-client' });
```

Comment:
I may be wrong here, but will this not work? When the CLI uses an access token to your account, but this is a demo api key that isn't linked to the same project is it?

Reply:
This is a very good point... I think I need to add a comment that if they don't use their own API key then they won't be able to view it in the dashboard or via CLI.